templates/zerver/help/enable-or-disable-pressing-enter-to-send.md

```diff
@@ -1,21 +1,22 @@
 # Press Enter to send
 
-By default, Zulip sends your message when you press enter (or return)
-on your keyboard. If you want to turn this off, simply uncheck the
-**Press Enter to send** box below the compose field:
+By default, Zulip sends your message when you press the **Send** button in
+the **Compose box**. If you would like to send your message by pressing
+the enter key, follow these steps:
 
-![Box checked](/static/images/help/enter-box-checked.png)
-
-You will now see a **Send** button below the compose field looking
-like this.
+1. Check the **Press Enter to send** box below the compose field:
 
 ![Box unchecked](/static/images/help/enter-box-unchecked.png)
 
-With the box unchecked, you will have to click the **Send** button to
-send your message. Pressing enter will no longer send your message.
+2. Now press the enter key to send your message.
 
-If you’d like to re-enable the press enter to send feature, just check
-the **Press Enter to send** box again, and you will be able to send
-your messages by pressing enter.
+!!! tip ""
+    If you’d like the enter key to not send messages again, just uncheck
+    the **Press Enter to send** box again, and you will be able to send
+    your messages by pressing the **Send** button like before.
+
+![Box checked](/static/images/help/enter-box-checked.png)
```

Comment:
You can remove and delete this image from the `static/help/images` folder, since the checked box difference is obvious